### ISSUE TITLE
Fatal Error: Call to undefined method setClassMapAuthoritative() in autoload_real.php

### ISSUE THREAD

**Issue by Source3Media** [awesomemotive/wp-mail-smtp#167]
Im trying to send test emails and this error popped up immediately.

I'm using WP Mail SMTP 1.2.5

## Current Behavior
As soon as I attempt to send a test email of any type, it automatically redirects me to an error page containing the following error:

```
Fatal error: Call to undefined method Composer\Autoload\ClassLoader::setClassMapAuthoritative() in /public_html/wp-content/plugins/wp-mail-smtp/vendor/composer/autoload_real.php on line 38.
```
When I first installed the plugin, there weren't any issues.

## Expected Behavior
My expectation is that the form would use my gmail(per my settings) to send out the information provided.

## Current System
My current system is using Siteground's shared hosting with Wordpress 4.9.4

## Screenshots (if appropriate)
![screenshot_12](https://user-images.githubusercontent.com/32966802/38213850-c87a4a44-3690-11e8-9913-547039cfa254.png)

**Comment by slaFFik** [comment]
@Source3Media 
You were able to install the plugin and configure it, correct? Which mailer have you selected? How have you installed the plugin on a server?

**Comment by Source3Media** [comment]
@slaFFik 
Yes, I was able to install and configure the plugin, all of which is set up to a GMail account and then just recently, it started to throw that error. I installed it directly through Wordpress Plugins Area onto my shared/reseller hosting on siteground.

**Comment by slaFFik** [comment]
@Source3Media 
Thanks for answers. So, basically, it worked for some time just ok, and recently broke, correct? 
Have you made any changes (like, moving the site elsewhere, activated/reactivated some other plugins)? 
Did Siteground notify you recently about server migrations or some server-related problems?
Was the PHP version on your site changed recently? Can you check it in Siteground cPanel for your this particular site?

**Comment by Source3Media** [comment]
@slaFFik As far as I am aware(I haven't gotten any notifications nor have I seen anything), there haven't been any changes to the server, I updated Wordpress about a month or so ago, but that didn't initially have any effect on the site.

**Comment by slaFFik** [comment]
Can you check it in Siteground cPanel for your this particular site the version of your PHP installed on a server?
Have you made any other changes - activated/deactivated some plugins etc?

**Comment by slaFFik** [comment]
You can try, but 5.6 is sufficient. 
7.2.3 may break something somewhere on your site, so do it with caution and be ready to rollback.
What about plugins? Are there any plugins that use Google API?

**Comment by Source3Media** [comment]
@slaFFik Yes, I have Google Analytics by Monster Insights installed on the site as well

**Comment by slaFFik** [comment]
This plugin is not the issue, that's for sure. Could you please share a list of active plugins here?

**Comment by Source3Media** [comment]
The active/installed plugins are as below
Advanced Custom Fields
All in One SEO
Broadcast
Content Timeline
Custom Post Type UI
Display Post Shortcodes
Font Resizer
Formidable Forms
Formidable Forms Pro
Formidable PayPal Standard
Formidable to Mailchimp
Google Analytics for Wordpress by Monster Insights
Jetpack
Limit Login Attempts
Media Library Categories
MomentoPress
Page Links to
Recent Posts Widget Extended
Subway
TinyMCE Adv
UpdraftPlus
User Role Editor
Wordfence + Wordfence Assist
WordPress Media Library Folders + Reset
WP Notification Bars
WP Statistics
wpMandrill
Youtube Embed

Most of these plugins are from my predecessors

**Comment by Source3Media** [comment]
@slaFFik I accidently tapped close

**Comment by slaFFik** [comment]
> https://wordpress.org/support/topic/fatal-error-2833/

**Comment by slaFFik** [comment]
@Source3Media 

> I checked all of the plugins, none of them got rid of the error message

So you can confirm, that when ALL plugins are deactivated except WP Mail SMTP - you still get this error? What about changing the theme back to some default, like 2015 (`twentyfifteen`)?

**Comment by Source3Media** [comment]
@slaFFik  I went as far back as I could without changing the appearance of my clients website, the error still showed up.

**Comment by slaFFik** [comment]
Might be related to #82
